tests/unit/commands/local/lib/test_local_api_service.py

```diff
@@ -80,50 +76,27 @@ def test_must_start_service(self,
     @patch("samcli.commands.local.lib.local_api_service.ApiProvider")
     @patch.object(LocalApiService, "_make_static_dir_path")
     @patch.object(LocalApiService, "_print_routes")
-    @patch.object(LocalApiService, "_make_routing_list")
+    @patch.object(ApiProvider, "_extract_routes")
     def test_must_raise_if_route_not_available(self,
-                                               make_routing_list_mock,
+                                               extract_routes,
                                                log_routes_mock,
                                                make_static_dir_mock,
                                                SamApiProviderMock,
                                                ApiGwServiceMock):
         routing_list = []  # Empty
 
-        make_routing_list_mock.return_value = routing_list
-
+        extract_routes.return_value = routing_list
+        SamApiProviderMock.extract_routes.return_value = routing_list
         SamApiProviderMock.return_value = self.api_provider_mock
         ApiGwServiceMock.return_value = self.apigw_service
 
         # Now start the service
         local_service = LocalApiService(self.lambda_invoke_context_mock, self.port, self.host, self.static_dir)
-
+        local_service.api_provider.api.routes = routing_list
         with self.assertRaises(NoApisDefined):
             local_service.start()
 
 
-class TestLocalApiService_make_routing_list(TestCase):
-
-    def test_must_return_routing_list_from_apis(self):
```

Comment:
Why was this tests removed? Do we have similar coverage if this was refactored out?

Reply:
An Api is now defined as a list of routes and it's attributes. All the function was doing was converting a list of Apis -> list of Routes, which isn't needed in the current model as it is directly passed through.